x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/layer_panel.tsx

```diff
@@ -249,11 +260,7 @@ export function LayerPanel(
                         }}
                         isValueEqual={isSameConfiguration}
                         label={columnLabelMap[accessor]}
-                        droppable={
-                          (dragging && !isDraggedOperation(dragging)) ||
-                          isFromCompatibleGroup ||
-                          isFromTheSameGroup
-                        }
+                        droppable={dragging && isDroppable}
```

Comment:
@mbondyra the previous logic was group-oriented, so I needed to change it so that it's only group-oriented when reorder or moving operations. Otherwise it's specific to each column.